Source/WebSocket.swift

```diff
@@ -355,6 +355,9 @@ open class WebSocket : NSObject, StreamDelegate {
                         return
                     }
                 }
+            } else {
```

Comment:
I believe adding this else branch will cause anyone not using SSL ciphers on an encrypted socket to fall into this, no? I think we would only want to fall into this if SSL ciphers are added? It isn't necessarily required to use SSL ciphers to connect to an encrypted socketed. 

P.S. I really like the descriptive error message. Nice work.